### DISCUSSION TITLE
[Feature] Support tags on uploadAsset and updateAssets endpoints

### DISCUSSION THREAD

**Discussion by marciopamplona** [immich-app/immich#11450] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The main objective is better identification of a group of assets by tagging them during a mass import via CLI. This way we can easily identify which import session that assets belongs to.
Sometimes I saw corrupted files in my mass imports, but I don't know exactly the origin of that file (I'm doing for someone else).

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by marciopamplona** [comment]
Related to #3518

**Comment by chriskoch** [comment]
I am currently working on a tool that downloads from iCloud and uploads to immich. This feature would be very helpful

**Comment by M4NH3X** [comment]
I don't see tags option in `uploadAsset` endpoint (https://immich.app/docs/api/upload-asset) and this is marked as Closed, can this be re-opened please?

**Comment by marvin78** [comment]
I would also like to know, why this is closed. This option for upload and update would be really helpfull for migration.
